torchvision/datapoints/_dataset_wrapper.py

```diff
@@ -98,7 +97,16 @@ def wrap_dataset_for_transforms_v2(dataset, target_keys=None):
             f"but got {target_keys}"
         )
 
-    return VisionDatasetDatapointWrapper(dataset, target_keys)
+    # Imagine we have isinstance(dataset, datasets.ImageNet). This will create a type with the name "WrappedImageNet" at
```

Comment:
I know it's the same but since the variable name is `wrapped_dataset_cls` this will be slightly more obvious.

```suggestion
    # Imagine we have isinstance(dataset, datasets.ImageNet). This will create a new class with the name "WrappedImageNet" at
```